client/main.ts

```diff
@@ -3,6 +3,7 @@
 'use strict';
 
 import * as React from 'react';
+
 import Hello from './Components/Hello';
 
-React.render(new Hello(/*{name: 'David'}*/), document.body);
+React.render(React.createElement(Hello, {name: 'David'}), document.body);
```

Comment:
New React API imposes this.
Either `React.createElement(Hello, props)` or `React.createElement('div', props)`. No more `new X` or `React.DOM.X`